app/routes/the-elevator-game.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+
+export default class TheElevatorGameRoute extends Route {
+   
+    model() {
+        return {
+            levels : Array.from(Array(5), (_, i) => i + 1),
```

Comment:
According to me, there is no need to add 1 explicitly because it is not defining its meaning here. it can be handled in the helper compute function by removing + 1 at line 15. what's your thoughts?